components/director/internal/domain/application/service.go

```diff
@@ -30,7 +31,7 @@ const (
 
 type repoCreatorFunc func(ctx context.Context, application *model.Application) error
 
-//go:generate mockery -name=ApplicationRepository -output=automock -outpkg=automock -case=underscore
+//go:generate mockery --name=ApplicationRepository --output=automock --outpkg=automock --case=underscore
```

Comment:
Either change all generated mocks to use the newer version of mockery or adjust your version to the one used before. Currently is it impossible to compile all mocks with one go command